### PR TITLE
Added archive family data functionality and updated search

### PR DESCRIPTION
**Pull request by brogers1500**
# Story
As a staff member I would like to archive data so that I can remove but save family information
# Acceptance Criteria
   1. Family can be archived from staff account
   3. Family can be unarchived from staff account
   4. Archived families can be view from the staff account

### CODE REVIEW

File: database/dbFamily.php [hunk 1]
```diff
@@ -335,4 +335,105 @@ function getHolidayMealBagData($family_id){
...
+        $families []= make_a_family2($row);
+    }
+    mysqli_close($connection);
+    return $families;
+
+}
+
+// Find family gets families based in criteria in parameters, it builds the query based on what is in it and what isn't 
+// More criteria can be added later
+function find_families($last_name, $email, $city, $archived){
```
Comment (itmacleod1995) [inline database/dbFamily.php:380]: I really like the way you construct the query in this function

File: database/dbFamily.php [hunk 2]
```diff
@@ -335,4 +335,105 @@ function getHolidayMealBagData($family_id){
...
+function unarchive_family($id) {
+    $query = "UPDATE dbFamily SET isArchived='0' WHERE id='$id'";
+    $connection = connect();
+    $result = mysqli_query($connection, $query);
+    $result = boolval($result);
+    mysqli_close($connection);
+    return $result;
+}
+
+function find_archived_families() {
```
Comment (itmacleod1995) [inline database/dbFamily.php:358]: I would just add comments/documentation to this function

File: findFamily.php [hunk 3]
```diff
@@ -53,13 +80,51 @@
         <h1>Search Family Account</h1>
```
Comment (itmacleod1995) [inline findFamily.php:80]: I would add somewhere some direction for the user. Say something like 'Please fill out at least one search field'

File: database/dbFamily.php [hunk 4]
```diff
@@ -335,4 +335,105 @@ function getHolidayMealBagData($family_id){
         $row = mysqli_fetch_assoc($res);
         return $row;
     }
+}
+
+function archive_family($id) {
```
Comment (itmacleod1995) [inline database/dbFamily.php:340]: Add comments

File: database/dbFamily.php [hunk 5]
```diff
@@ -335,4 +335,105 @@ function getHolidayMealBagData($family_id){
...
+    $query = "UPDATE dbFamily SET isArchived='1' WHERE id='$id'";
+    $connection = connect();
+    $result = mysqli_query($connection, $query);
+    $result = boolval($result);
+    mysqli_close($connection);
+    return $result;
+}
+
+function unarchive_family($id) {
```
Comment (itmacleod1995) [inline database/dbFamily.php:349]: Add comments

File: database/dbFamily.php (hunk 4 above)
Comment (brogers1500) [reply database/dbFamily.php:340]: I'll add some

File: database/dbFamily.php (hunk 5 above)
Comment (brogers1500) [reply database/dbFamily.php:349]: I'll add some comments here

**Review by itmacleod1995 — APPROVED** [review]
Looks good to me

**Pull request merged**